docs/modules/operation/pages/kafka-producer/kafka-producer.adoc

```diff
@@ -45,14 +45,15 @@ If the node is not associated with both a foreign source and foreign ID, then th
 
 == Topologies
 
-The Kafka Producer listens for changes made to the current set of topologies (bridge, CDP, ISIS, LLDP, and OSPF) and forwards the resulting messages to Kafka topics. A topic is defined either for Topology Vertex Update Message and another topic is defined for Topology Edge Update Message.
+The Kafka Producer listens for changes made to the current set of topologies (bridge, CDP, ISIS, LLDP, and OSPF) and forwards the resulting messages to Kafka topics.
+A topic is defined either for Topology Vertex Update Message and another topic is defined for Topology Edge Update Message.
```

Comment:
```suggestion
A topic is defined for Topology Vertex Update Message and another topic is defined for Topology Edge Update Message.
```

or is it "either/or"? -- "A topic is defined either for Topology Vertex Update Message or Topology Edge Update Message."

Reply:
@agalue Can you confirm?

Reply:
That'd be a question for @cgorantla as I've never seen that part of the Kafka Producer working.

Reply:
There is a "topologyVertexTopic" property defined in the Blueprint, but it is not used.

We only send "Topology Edge" messages, and these include the details about the associated vertices.

Reply:
So, something like removing this line and modifying the line above to:
> The Kafka Producer listens for changes made to the current set of topologies (bridge, CDP, ISIS, LLDP, and OSPF) and forwards the resulting messages to the topologyEdgeTopic Kafka topic.